app/uk/gov/hmrc/economiccrimelevyreturns/services/EnrolmentStoreProxyService.scala

```diff
@@ -29,6 +29,9 @@ class EnrolmentStoreProxyService @Inject() (enrolmentStoreProxyConnector: Enrolm
   ec: ExecutionContext
 ) {
 
+  // TODO This method could return a "maybe" registration date (in the future).
+  //      No registration date would mean the user is not yet registered
+  //
```

Comment:
You don't need to change anything here - if we are calling this API it is because we assume they are registered - please take a look at AuthorisedAction which is where the logic is to determine whether someone is registered or not